content/_includes/post.macros.njk

```diff
@@ -162,24 +178,27 @@ params:
   collections:
     type: 11ty collections object
     default: none
-  feature:
-    type: boolean | 'large'
+  loop:
+    type: Nunjucks `loop` object -- https://mozilla.github.io/nunjucks/templating.html#for
     default: none
   override_feature:
     type: boolean
     default: 'false'
+  override_large:
+    type: boolean
+    default: 'false'
 #}
 {% macro item(
   post,
   collections=none,
   loop=none,
-  override_feature=false
+  override_feature=false,
+  override_large=false
 ) %}
   {%- set type = post.data.tags | pageType -%}
   {%- set card = post.data.card or none -%}
-  {%- set card = 'feature' if (card == 'feature') else card -%}
-  {%- set card = card or 'feature' if type.feature else card -%}
-  {%- set card = none if override_feature else card -%}
+  {%- set card = 'feature' if (override_large and card === 'large') else card -%}
+  {%- set card = none if (override_feature and card === 'feature') else card -%}
```

Comment:
Adjusted the logic here so that we can set `override_feature` and `override_large` independently. If `override_large` is used without `override_feature`, `large` cards are converted to `feature` cards.

Reply:
Which means that anything marked as `card: large` will show up as large in the post lists, but it will look like `card: featured` on the homepage... Is that the intention of this?

Reply:
Correct!